src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -324,16 +339,21 @@ public Object apply(WebDriver webDriver) {
     });
   }
 
-  public String getGptParams(String slotName, String attr) {
-    WebElement
-        adsDiv =
-        driver.findElement(
-            By.cssSelector("#"+slotName+" [" + attr + "]"));
-    return adsDiv.getAttribute(attr);
+  public String getSlotAttribute(String slotName, String attr) {
+    try {
+      WebElement
+          adsDiv =
+          driver.findElement(
+              By.cssSelector("#"+slotName+" [" + attr + "]"));
+      return adsDiv.getAttribute(attr);
+    } catch (NoSuchElementException elementNotFound) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)